src/main/java/net/fabricmc/example/mixin/ExampleMixin.java

```diff
@@ -1,15 +1,25 @@
 package net.fabricmc.example.mixin;
 
+import net.minecraft.client.gui.screen.Screen;
 import net.minecraft.client.gui.screen.TitleScreen;
 import org.spongepowered.asm.mixin.Mixin;
 import org.spongepowered.asm.mixin.injection.At;
 import org.spongepowered.asm.mixin.injection.Inject;
 import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
 
 @Mixin(TitleScreen.class)
-public class ExampleMixin {
-	@Inject(at = @At("HEAD"), method = "init()V")
+public abstract class ExampleMixin extends Screen {
+	private MixinTitleScreen(Text title) {
```

Comment:
protected i guess?

Reply:
private works and makes it so someone doesn't accidentally initialize the mixin class (if you are extending ExampleMixin then yes protected is the only way to compile).

Well then again it shouldn't happen anyways in the first place.

Reply:
sure, do whatever you like here, i guess